client/tests/webdriver/specs/editLocation.spec.js

```diff
@@ -1,50 +1,62 @@
+import CreateNewLocation from "../pages/location/createNewLocation.page"
 import EditLocation from "../pages/location/editLocation.page"
+import ShowLocation from "../pages/location/showLocation.page"
+import { LOCATION_COORDS, LOCATION_NAME, NEW_COORDS } from "./locationUtils"
+
+describe("When editing a location", () => {
+  it("Should create a new location first", () => {
```

Comment:
Well, for me this would mean that you are trying to test that "When editing a location Should create a new location first". But this is not the case, you create a location first just to be able to test something further.
Maybe you can use a beforeEach in the test and do there the creation.